gflownet.py

```diff
@@ -486,6 +489,7 @@ def __init__(
         self.oracle_period = args.gflownet.oracle.period
         self.oracle_nsamples = args.gflownet.oracle.nsamples
         self.oracle_k = args.gflownet.oracle.k
+        self.reward_clip_min = 1e-5
```

Comment:
I believe this is not used

Reply:
But rewards come clipped from the environment (see [here](https://github.com/alexhernandezgarcia/gflownet/blob/main/gflownetenv.py#L110)). Do we need double clipping?

Reply:
Oh right, sorry missed this. Yeah no, double clipping does not make sense. I'll delete this